pysat/_instrument.py

```diff
@@ -890,7 +890,7 @@ def _load_data(self, date=None, fid=None):
                                        date.strftime('%D')))
                 '''
                 output_str = ' '.join(('Returning', output_str, 'data for',
-                                       date.strftime('%j')))
+                                       date.strftime('%x')))
```

Comment:
Are we interested in date that looks like "09/29/2018" here? %j gives day of the year as a 3-digit number with zero padding if needed.

Reply:
The purpose is just to have an easily human readable date in output.  Since month/day/year is more commonly encountered than DOY, I defaulted to that when solving the merge conflicts.